x-pack/plugins/monitoring/server/routes/api/v1/logstash/pipeline.js

```diff
@@ -75,13 +76,18 @@ export function logstashPipelineRoute(server) {
       }
 
       try {
-        const [pipeline, vertex] = await Promise.all(promises);
+        const [pipeline, vertex] = await Promise.all(promises).catch((error) => {
+          throw error;
```

Comment:
is the error logged upstream or somewhere else ? if yes we can remove this catch block, otherwise that would be a good place to add it :)

Reply:
ah the outer catch is probably a better fit

Reply:
I've removed the `.catch` on the `Promise.all` since it's superfluous (was just trying to be extra explicit but the outer `catch` handles everything anyway). And I've added a server log for us in that outer `catch`.

![Screenshot 2022-05-12 at 10 38 30](https://user-images.githubusercontent.com/471693/168042605-b4209a0a-3097-4137-bd63-aa3aaa42c3ac.png)